handlers/aws/replay_trigger.py

```diff
@@ -15,19 +15,20 @@ class ReplayedEventReplayHandler:
     def __init__(self, replay_queue_arn: str):
         self._replay_queue_arn = replay_queue_arn
         self._failed_event_ids: list[str] = []
-        self._event_ids_with_receipt_handle: dict[str, str] = {}
+        self._events_with_receipt_handle: dict[str, str] = {}
```

Comment:
changes if this file address the first bugfix:
- proper handling of multiple outputs in the replay queue handler

the event id is the same across multiple outputs, when the replay handler of the replay queue applies its logic in order to understand if the events were successfully replied we must rely on a event _unique_ id suffixed by the output type